spec/truefi2/scripts/BorrowingMutex.sh

```diff
@@ -1,3 +1,2 @@
 certoraRun contracts/truefi2/BorrowingMutex.sol \
-  --verify BorrowingMutex:spec/truefi2/BorrowingMutex.spec \
-  --rule_sanity
```

Comment:
As mentioned in #1130, the `rule_sanity` checks are giving a false-negative error right now, hence temporarily removed.